src/components/block/index.ts

```diff
@@ -690,11 +685,47 @@ export default class Block {
         pluginsContent = this.toolInstance.render();
 
     contentNode.appendChild(pluginsContent);
-    wrapper.appendChild(contentNode);
+
+    let wrappedContentNode: HTMLElement = contentNode;
+
+    [...this.tunesInstances.values(), ...this.defaultTunesInstances.values()]
+      .forEach((tune) => {
+        if (_.isFunction(tune.wrap)) {
+          wrappedContentNode = tune.wrap(wrappedContentNode);
```

Comment:
do we need to add try-catch statement here? To avoid rendering problems in case of an error on the tune side?

Reply:
same thing at the `save()` method